common/readfromfile.go

```diff
@@ -11,6 +11,9 @@ import (
 // A valid string will always be returned, regardless of whether an error occurred.
 func ReadFromFile(s string) (string, error) {
 	info, err := os.Stat(s)
+	if err != nil {
+		return s, err
+	}
 	if os.IsNotExist(err) {
```

Comment:
Doesn't this check now become redundant?

Reply:
true, removed it